.config-schema.json

```diff
@@ -435,24 +435,70 @@
             "type": "object",
             "properties": {
                 "enabled": {
-                    "description": "enable throttling when request's number of dispatches is high",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_ENABLED instead. enable throttling when request's number of dispatches is high",
                     "type": "bool",
                     "default": "false",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
                 },
                 "frequency": {
-                    "description": "the frequency period that the deprioritized throttling queue is evaluated. A higher value will result in more aggressive throttling",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_FREQUENCY instead. the frequency period that the deprioritized throttling queue is evaluated. A higher value will result in more aggressive throttling",
                     "type": "duration",
                     "default": "10µs",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_FREQUENCY"
                 },
                 "threshold": {
-                    "description": "define the number of recursive operations to occur before getting throttled",
+                    "description": "DEPRECATED: Use OPENFGA_CHECK_DISPATCH_THROTTLING_THRESHOLD instead. define the number of recursive operations to occur before getting throttled",
```

Comment:
Elsewhere in our descriptions in this json-schema we strive to use our config name format instead of ENV variables. Can we please stay consistent with that or change the others elsewhere?

e.g. this would change to
```
"description": "DEPRECATED: Use 'checkDispatchThroltting.threshold' instead.
```